db/migrate_cards/20151223194824_import_tweaks_bug_fixes.rb

```diff
@@ -0,0 +1,8 @@
+# -*- encoding : utf-8 -*-
+
+class ImportTweaksBugFixes < Card::Migration
+  def up
+    import_json "tweaks_bug_fixes.json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.